tests/foreman/cli/test_ostreebranch.py

```diff
@@ -19,138 +19,152 @@
 import random
 
 import pytest
+from nailgun import entities
 
 from robottelo.cli.contentview import ContentView
 from robottelo.cli.factory import make_content_view
 from robottelo.cli.factory import make_org_with_credentials
 from robottelo.cli.factory import make_product_with_credentials
 from robottelo.cli.factory import make_repository_with_credentials
-from robottelo.cli.factory import make_user
 from robottelo.cli.ostreebranch import OstreeBranch
 from robottelo.cli.repository import Repository
 from robottelo.config import settings
-from robottelo.constants.repos import FEDORA27_OSTREE_REPO
+from robottelo.constants.repos import OSTREE_REPO
 from robottelo.decorators.host import skip_if_os
-from robottelo.test import CLITestCase
+
+
+@pytest.fixture(scope='module')
+def ostree_user_credentials():
+    password = 'password'
+    user = entities.User(admin=True, password=password).create()
+    return user.login, password
+
+
+@pytest.fixture(scope='module')
+@pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
```

Comment:
This was discussed, and I believe was already suggested to be updated, but an approval has been made with this active.

AFAIK, you cannot mark fixtures like this and have it apply to the tests.  Can you demonstrate that the tests will actually skip when this mark is applied to the fixture instead of to the test functions, module, or class?

Reply:
If you want to apply this without adding it to every function in the module, define this a the module level:

`pytestmark = [pytest.mark.skipif(yada yada yada)]`

Reply:
yea definitely, I just didn't think... :expressionless: